environment_test.go

```diff
@@ -0,0 +1,33 @@
+package driver_test
```

Comment:
Why is this file needed

Reply:
I want the test to be focused on the thing it tests. So I moved the setup/teardown code to this file.

Reply:
Since all test code is in the test folder and setup is also in there, I don't want to pollute the root folder with separate test code.